### PR TITLE
Some Ceres fixes

### LINKED ISSUE #2260: Ceres texture
Open dawn profile and go to Ceres. For some reason, the texture does not show up and the planet just looks like a gray marble

### LINKED ISSUE #2261: Move Ceres, Dawn and Vesta to dwarf planets folder instead of dawn missions folder in assets?
The natural place to search for these is under dwarf planets, not necessarily under the dawn mission

Experienced by user in support slack: https://openspacesupport.slack.com/archives/C65DVB1EY/p1664684240963699

### PR DESCRIPTION
**Pull request by Roxeena**
A few fixes regarding the dwarf planet Ceres

- Adds a downsampled texture for Ceres (closes #2260)
- Moves the Ceres asset to the dwarf_planets folder (closes #2261)
- Separates the kernels for Ceres and Vesta from the kernels for dawn

The streamable layer with the full-resolution texture is not included in this PR (yet).

The data will be uploaded soon.

### CODE REVIEW

**Comment by alexanderbock** [comment]
Files are uploaded and WMS layer is added :+1:

**Review by alexanderbock — APPROVED** [review]

File: data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo.asset [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  openspace.globebrowsing.deleteLayer(globeIdentifier, "ColorLayers", layer)
+end)
+
+asset.export("layer", layer)
+
+
+asset.meta = {
+  Name = "LAMO [Sweden]",
+  Version = "1.0",
+  Description = [[]],
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo.asset:24]: Add description, author, url, etc please :)

File: data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo_local.asset [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  openspace.globebrowsing.deleteLayer(globeIdentifier, "ColorLayers", layer)
+end)
+
+asset.export("layer", layer)
+
+
+asset.meta = {
+  Name = "LAMO [Local]",
+  Version = "1.0",
+  Description = [[]],
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo_local.asset:31]: same here

File: data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo.asset [hunk 3]
```diff
@@ -0,0 +1,28 @@
+local globeIdentifier = asset.require("../../ceres").Ceres.Identifier
+
+local layer = {
+  Identifier = "LAMO_",
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/dwarf_planets/ceres/layers/colorlayers/lamo.asset:4]: Why the underscore in the identifier?

**Review by GPayne — APPROVED** [review]
My only comment is actually not something you modified, but since you're already into this asset maybe you could address it:
dawn/dawn.asset has commented-out code at lines 111 and 140, explaining the lines are disabled because SPICE CK kernels are not present, but the file does contain CK kernels.

**Comment by alexanderbock** [comment]
I merged it as it is to be able to make the release candidate branch

**Pull request merged**